kdc/kdc_locl.h

```diff
@@ -79,6 +79,7 @@ struct astgs_request_desc {
     EncKDCRepPart ek;
 
     /* PA methods can affect both the reply key and the session key (pkinit) */
+    int validated_pa_type;
```

Comment:
Can you make this

`unsigned int validated_pa_type : 1`

and put it immediately above `armor_crypto`, this will be more consistent when we merge lukeh/gss-preauth-apple

Reply:
Ignore that, sorry, I didn't realise this was not a boolean. In which case, this should be `krb5_preauthtype pa_type_validated`.

Reply:
I've changed the type